ssid.txt

```diff
@@ -0,0 +1 @@
+MTAwMDI3ODY2OXxleUowZVhBaU9pSktWMVFpTENKaGJHY2lPaUpJVXpVeE1pSjkuZXlKbGVIQWlPakUyTVRjeU5EUTBNamNzSW1saGRDSTZNVFl4TWpBMk1EUXlOeXdpYzNWaUlqb2lNVEF3TURJM09EWTJPU0lzSW1WdFlXbHNJam9pWldabWFXTnBaVzVqZVc1MWRITkFaMjFoYVd3dVkyOXRJbjAuR2plQWpvdWhpa1ZUY0JDdlB6ekw0aFlSZ2YwM2RpWG9idnR0d2xKT3BkWXVMNUg0Ny0zYXVOWVJtSTFnZXpCOUREeFlVcUdBc19rMXhUZnBlZ2VIZ1E=
```

Comment:
Do not commit your Kijiji cookie!
Committing your private cookie gives anyone access to your Kijiji session. You should probably logout of your current Kijiji session to invalidate this cookie.

Reply:
oups! :)